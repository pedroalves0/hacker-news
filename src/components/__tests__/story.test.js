import React from 'react';
import { shallow } from 'enzyme';
import Story from '../story';

describe('components/story', () => {
    test('should render correctly', () => {
        const wrapper = shallow(
            <Story
                by={'author'}
                time={1512157581}
                title={'title'}
                url={'http://www.url.com/url/url/'}
            />
        );

        expect(wrapper).toMatchSnapshot();
    });
});