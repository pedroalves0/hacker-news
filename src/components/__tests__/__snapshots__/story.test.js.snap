// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/story should render correctly 1`] = `
<div
  className="story"
>
  <div
    className="story-title"
  >
    <a
      className="story-title-link"
      href="http://www.url.com/url/url/"
    >
      title
    </a>
    <span
      className="story-title-hostname"
    >
       (
      <a
        href="http://www.url.com"
      >
        url.com
      </a>
      )
    </span>
  </div>
  <div
    className="story-info"
  >
    <span>
      by 
      <span
        className="t-ital"
      >
        author
         - 
      </span>
    </span>
    <TimeAgo
      component="time"
      date={2017-12-01T19:46:21.000Z}
      formatter={[Function]}
      live={true}
      maxPeriod={Infinity}
      minPeriod={0}
      now={[Function]}
    />
  </div>
</div>
`;
